templates/user/username_link.html

```diff
@@ -1,7 +1,14 @@
-{% if user.moderator %}
+{% if user.id == streamer.id %}
+<div class="pui badge broadcaster"></div>
+{% elif user.moderator %}
```

Comment:
this should not be in here

Reply:
There is no record of the streamer being in the user models file. How would the bot import `user.streamer` if it isn't defined?

Reply:
I thought it was said on stream that the broadcaster badge would be added separately. I want to merge this too, adding change after change after change infinitely delays the PR. Please remove the two marked changes and then we can get this PR merged. We can talk about this kind of stuff in a separate Pull request

Reply:
No, pajlada said if it's badge-related then it can remain in this PR. For the stuff that's related to the https://github.com/pajbot/pajbot/pull/950 - it can be committed in that PR. The broadcaster badge on the userpage has nothing to do with the clip command pr.